src/main/webapp/js/communication/geppettoJupyter/PythonControlledCapability.js

```diff
@@ -141,12 +170,17 @@ define(function (require) {
                                 break;
                             case 'dict':
                                 newValue = JSON.parse(newValue)
+                                break;
+                            case 'func':
+                                if (newValue=='') {
+                                    newValue = 1
```

Comment:
When the value is empty do we want to set it to 1? Are you sure about this? Just asking...